cms/static/sass/elements/_controls.scss

```diff
@@ -295,12 +295,12 @@
 
       // CASE: right to left layout
       @include rtl {
-       background: transparent url("../images/drag-handles.png") no-repeat left center;
+       background: transparent url($static-path + "/images/drag-handles.png") no-repeat left center;
```

Comment:
Please see [my comments about the formatting for passing this path variable](https://github.com/edx/edx-platform/pull/8271/files#r37541733) into rules like this.